src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -4277,3 +4277,19 @@ export enum KeywordRecognitionStatus {
 }
 
 //#endregion
+
+//#region InlineEdit
+
+export class InlineEdit implements vscode.InlineEdit {
+	constructor(
+		public readonly text: string,
+		public readonly range: Range,
```

Comment:
This does not properly implement the interface, as the interface is not readonly.

Reply:
Updated the interface to make `text` and `range` read-only.